pkg/etw/newprovider.go

```diff
@@ -1,4 +1,4 @@
-// +build amd64 arm64 386
+// +build windows amd64 arm64 386
```

Comment:
Do we want windows to be a ANDed instead of an ORed here? Seems weird to have an OS OR'd with architecture types.

Reply:
Same for some of the other files in this PR

Reply:
Alternatively, I think renaming the file to `pkg/etw/newprovider_windows.go` would work (and could keep the `// +build` comments less complicated)

Reply:
We can have multiple build constraints per file, and each line is effectively ANDed together. So here we could do:

```
// +build windows
// +build amd64 arm64 386
```

This would be my preference, as I think it's slightly simpler than renaming the files. :)

Reply:
Good idea, thank you.